code/game/objects/items/devices/mat_synth.dm

```diff
@@ -34,9 +34,10 @@
 							 "floor tiles" = /obj/item/stack/tile/plasteel,
 							 "metal rods" = /obj/item/stack/rods)
 
-/obj/item/device/material_synth/robot/New()
+/obj/item/device/material_synth/robot/New() //We have to do this during New() because BYOND can't pull a typesof() during compile time.
 	. = ..()
-	can_scan = existing_typesof(/obj/item/stack/sheet) - list(/obj/item/stack/sheet/mineral/clown, /obj/item/stack/sheet/mineral/phazon)
+	if(!istype(src, /obj/item/device/material_synth/robot/mommi))
```

Comment:
lrn2OOP

(Just re-add them in the MoMMI one's `New()`, or add a string deal like storage items, or something)

Reply:
I had it working like that at first, but didn't like that the "non-borg can scan list" was copypasted twice that way.